docs/_guides/setup-pouchdb.md

```diff
@@ -49,7 +49,7 @@ Then in your `index.html`:
 Add this to your `index.html`:
 
 ```html
-<script src="//cdn.jsdelivr.net/pouchdb/{{site.version}}/pouchdb.min.js"></script>
+<script src="//cdn.jsdelivr.net/pouchdb@{{site.version}}/dist/pouchdb.min.js"></script>
```

Comment:
There's missing `/npm/` at the beginning - should be `//cdn.jsdelivr.net/npm/pouchdb@{{site.version}}/dist/pouchdb.min.js`

Reply:
Fixed

Reply:
This still isnt fixed, it does automatically forward but if we are fixing this may as well fix it to the corrected url